lua/dap/session.lua

```diff
@@ -1623,7 +1623,7 @@ end
 --- Initialize the debug session
 ---@param config Configuration
 function Session:initialize(config)
-  vim.schedule(repl.clear)
+  vim.schedule(repl.clear_output)
```

Comment:
```suggestion
  vim.schedule(repl.clear)
```

Reply:
Is it possible to add an option to control whether to clear the history when start a new session?